build/_k-restore.shade

```diff
@@ -4,7 +4,7 @@ k-restore
     Restores nuget packages required for k projects. Downloads and executes k sdk tools.
```

Comment:
These should be changed when we remove k.cmd and friends. There's a work item in the other repo.